.github/workflows/golangci-lint.yaml

```
@@ -0,0 +1,40 @@
+---
+name: golangci-lint
+
+on:
+  pull_request:
+    branches: ['*']
```

Strict YAML notation is preferable 
```suggestion
    branches: 
    - '*'
```

---

Updated in latest commit